presto-docs/src/main/sphinx/sql/revoke.rst

```diff
@@ -0,0 +1,53 @@
+======
+REVOKE
+======
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    REVOKE [ GRANT OPTION FOR ]
+    (privilege [, ...] | ALL PRIVILEGES)
+    ON [ TABLE ] table_name FROM grantee
+
+Usage of the term ``grantee`` denotes both, users and roles.
+
+Description
+-----------
+
+Revokes the specified privileges from the specified grantee.
+
+Specifying ``ALL PRIVILEGES`` revokes ``DELETE, INSERT`` and ``SELECT`` privileges.
+
+Specifying ``PUBLIC`` revokes privileges from all grantees.
```

Comment:
I checked SQL standard, and revoking privileges from `PUBLIC` role is listed. The `REVOKE` code in Presto invokes the underlying hive metastore API method, so the behavior is dependent on that implementation.